nats-server/routez.yml

```diff
@@ -18,3 +18,44 @@ observation:
     tag:
       - name: nats.server.id
         value: json:/routez $.server_id
+
+  - name: routez_route
+      metricNamespace: nats
+      path: $.routes[?(@.name=${routeName}})]
+
+      metric:
+        - name: server.routes.size.pending
```

Comment:
should name be `server.route.size.pending`?

Reply:
I change to just server.route.pending